docs/commands/got.md

```diff
@@ -6,9 +6,11 @@ The `got` command optionally takes function names and filters the output display
 matching functions.
 
 ```text
-gef➤ got
+gef➤ got [--all] [--filters]
```

Comment:
What is `--filters`? I don't see it in the code, care to elaborate?
If it's supposed to be in the code, also document it

Reply:
...that's embarrassing.  Fingers were moving, brain was not.  That was intended to be only 'filters'.

Reply:
I missed this too!